### ISSUE TITLE
Extra whitespace after "#{}" interpolation

### ISSUE THREAD

**Issue by doyoe** [sass/libsass#1722]
```
background-position: 0 -#{map-get($score, item-height)};
```

Compile to: 

```
background-position: 0 - 0.12rem;
```

There was a space in code.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/607#event-463586014

**Comment by xzyfer** [comment]
Fixed by #1739.

This will be in 3.3.3. Subscribe to https://github.com/sass/libsass/issues/1808 for updates on that release.

**Comment by xzyfer** [comment]
Spec activated https://github.com/sass/sass-spec/pull/657
